component/Authenication/Login/Login.html

```diff
@@ -40,20 +40,20 @@
             </div>
         </nav>
 
-        <div class="auth_container flex">
+        <section class="auth_container flex">
             <div class="auth shadow flex">
                 <div class="content_subheading_heading auth_head">Login</div>
                 <div>
-                    <label for=" fname"><b>Email address:</b></label>
+                    <label for="email"><b>Email address:</b></label>
                     <div class="nav_input auth_input">
-                        <input class="input input_wide shadow" type="text" placeholder="prakash@gmail.com">
+                        <input class="input input_wide shadow" id=email" type="text" placeholder="prakash@gmail.com">
                     </div>
                 </div>
 
                 <div>
-                    <label for=" fname"><b>password:</b></label>
+                    <label for="password"><b>password:</b></label>
                     <div class="nav_input auth_input">
-                        <input class="input input_wide shadow" type="password" placeholder="******">
+                        <input class="input input_wide shadow" id="password" type="password" id placeholder="******">
```

Comment:
I  think there is an extra `id` keyword here, after the `type` attribute. Probably a typo.